lib/browser/rpc-server.js

```diff
@@ -476,10 +481,22 @@ ipcMain.on('ELECTRON_BROWSER_SANDBOX_LOAD', function (event) {
   }
 })
 
-ipcMain.on('ELECTRON_BROWSER_GET_LAST_WEB_PREFERENCES', function (event) {
+const handleReturnValue = function (event, code) {
   try {
-    event.returnValue = [null, event.sender.getLastWebPreferences()]
+    event.returnValue = [null, code()]
   } catch (error) {
-    event.returnValue = [errorUtils.serialize(error)]
+    event.returnValue = [errorUtils.serialize]
```

Comment:
You probably wanted to call `[errorUtils.serialize(error)]`.

Reply:
of course :)

Reply:
fixed